ui/perfherder/Validation.jsx

```diff
@@ -33,6 +33,7 @@ const withValidation = ({ requiredParams }, verifyRevisions = true) => (
         selectedTimeRange: null,
         framework: null,
         validationComplete: false,
+        performanceTags: [],
```

Comment:
I'm not sure this is necessary. At least not at this moment.

Reply:
Indeed, this was not necessary. Thank you!